src/Services/Database/VerificationService.php

```diff
@@ -118,7 +122,8 @@ public function generate(): array
     public function qr(string $verificationId, string $code, string $callback, int $size = 512): string
     {
         $encodedCallback = base64_encode($callback);
-        $deepLink = config('enjin-platform.deep_links.proof') . "{$verificationId}:{$code}:{$encodedCallback}";
+        $encodedCode = base64_encode('espr:' . $code);
+        $deepLink = config('enjin-platform.deep_links.proof') . "{$verificationId}:{$encodedCode}:{$encodedCallback}";
 
         return "https://chart.googleapis.com/chart?chs={$size}x{$size}&cht=qr&chl={$deepLink}";
```

Comment:
We should port the `QrAdapterInterface` over from Beam so we can use it here (and anywhere else that needs to generate QRs).